common/src/main/java/org/opensearch/ml/common/transport/update/MLUpdateModelCacheNodesRequest.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.ml.common.transport.update;
+
+import lombok.Getter;
+import org.opensearch.action.support.nodes.BaseNodesRequest;
+import org.opensearch.cluster.node.DiscoveryNode;
+import org.opensearch.core.common.io.stream.StreamInput;
+import org.opensearch.core.common.io.stream.StreamOutput;
+import java.io.IOException;
+
+public class MLUpdateModelCacheNodesRequest extends BaseNodesRequest<MLUpdateModelCacheNodesRequest> {
+
+    @Getter
+    private String modelId;
+    @Getter
+    private boolean predictorUpdate;
```

Comment:
Add more comment or make this variable more readable?

Reply:
It will be removed.